lib/parsers/llvm-pass-dump-parser.ts

```diff
@@ -472,35 +462,36 @@ export class LlvmPassDumpParser {
             lineFilters = lineFilters.concat(this.debugInfoLineFilters);
         }
 
-        // Filter junk
-        // prettier-ignore
-        const idxAfterHeaders = this.findLastIndex(
-            ir,
-            line => (line.text.match(this.irDumpHeader) || line.text.match(this.machineCodeDumpHeader)) !== null);
-
-        return ir
-            .slice(idxAfterHeaders)
-            .filter(line => filters.every(re => line.text.match(re) === null)) // apply filters
-            .map(_line => {
-                let line = _line.text;
-                // eslint-disable-next-line no-constant-condition
-                while (true) {
-                    let newLine = line;
-                    for (const re of lineFilters) {
-                        newLine = newLine.replace(re, '');
-                    }
-                    if (newLine === line) {
-                        break;
-                    } else {
-                        line = newLine;
+        return (
```

Comment:
Why the parenthesis here? Prettier change?

Reply:
It is indeed a prettier change. I tried some things to get it to make a different choice but it was quite insistent on using those parentheses.